Controller:
  hero_chassis_controller:
    type: hero_chassis_controller/HeroChassisController
    left_front_wheel_joint:
      joint: front_left_joint_
#      pid: {p: 100.0, i: 0.01, d: 10.0}

    right_front_wheel_joint:
      joint: front_right_joint_
#      pid: {p: 100.0, i: 0.01, d: 10.0}

    left_back_wheel_joint:
      joint: back_left_joint_
#      pid: {p: 100.0, i: 0.01, d: 10.0}

    right_back_wheel_joint:
      joint: back_right_joint_
#      pid: {p: 100.0, i: 0.01, d: 10.0}

    pid: {p: 100.0, i: 0.01, d: 10.0}


  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

#  left_front_wheel_joint_controller:
#    type: velocity_controllers/joint_velocity_controller
#    joint: left_front_wheel_joint
#    pid: {p: 100.0, i: 0.01, d: 10.0}
#
#  right_front_wheel_joint_controller:
#    type: velocity_controllers/joint_velocity_controller
#    joint: right_front_wheel_joint
#    pid: {p: 100.0, i: 0.01, d: 10.0}
#
#  left_back_wheel_joint_controller:
#    type: velocity_controllers/joint_velocity_controller
#    joint: left_back_wheel_joint
#    pid: {p: 100.0, i: 0.01, d: 10.0}
#
#  right_back_wheel_joint_controller:
#    type: velocity_controllers/joint_velocity_controller
#    joint: right_back_wheel_joint
#    pid: {p: 100.0, i: 0.01, d: 10.0}